### PR TITLE
BIOWDL-316: Add documentation to repository

### PR DESCRIPTION
**Pull request by JasperBoom**
### Checklist
- [X] Pull request details were added to CHANGELOG.md

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+---
+layout: default
+title: Home
+---
+
+This pipeline can be used to process RNA sequenced by either the Pacific Biosciences or Oxford Nanopore sequencer, starting from FastQ files. It will perform mapping to a reference genome (using minimap2), INDEL/mismatch and noncanonical splice junction correction (using TranscriptClean) and identify and count known and novel genes/transcripts (using TALON).
```
Comment (DavyCats) [inline docs/index.md]: Please split this up over a few lines.

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+described below, but additional inputs are available.
+A template containing all possible inputs can be generated using
+Womtool as described in the
+[WOMtool documentation](http://cromwell.readthedocs.io/en/stable/WOMtool/).
+
+```json
+{
+    "Pipeline.sampleConfigFile": "A sample configuration file (see below).",
+    "Pipeline.outputDirectory": "The path to the output directory.",
+    "Pipeline.annotationGTF": "GTF annotation containing genes, transcripts, and edges.",
```
Comment (DavyCats) [inline docs/index.md:34]: edges or exons?

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    "Pipeline.genomeBuild": "Name of genome build that the GTF file is based on (ie hg38).",
+    "Pipeline.annotationVersion": "Name of supplied annotation (will be used to label data).",
+    "Pipeline.referenceGenome": "Reference genome fasta file.",
+    "Pipeline.sequencingPlatform": "The sequencing platform used to generate long reads.",
+    "Pipeline.talonConfigFile": "TALON specific configuration file",
+    "Pipeline.pipelineRunName": "A short name to distinguish a run.",
+    "Pipeline.dockerImagesFile": "A file listing the used docker images.",
+    "Pipeline.sampleWorkflow.presetOption": "This option applies multiple options at the same time to minimap2."
```
Comment (DavyCats) [inline docs/index.md]: What kind of value am I suppose to put here as user? A boolean, a string?

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+For more advanced configuration of docker or singularity please check
+the [cromwell documentation on containers](
+https://cromwell.readthedocs.io/en/stable/tutorials/Containers/).
+
+Images from [biocontainers](https://biocontainers.pro) are preferred for
+biowdl pipelines. The list of default images for this pipeline can be
+found in the default for the `dockerImages` input.
+
+### Output
+The workflow will output mapped reads by minimap2 in a .sam file, a cleaned .sam file and log information from TranscriptClean, a database containing transcript information together with a log file of read/transcript comparison and a abundance plus summary file of the database.
```
Comment (DavyCats) [inline docs/index.md]: Please split this up over a few lines.

**Review by DavyCats — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 2 above)
Comment (JasperBoom) [reply docs/index.md:34]: I took this line from the TALON documentation.
Edges can be either exons or introns in the TALON database, these are stored in the "edge" table.

**Review by DavyCats — APPROVED** [review]

**Pull request merged**